python/lsst/ctrl/mpexec/cmdLineFwk.py

```diff
@@ -570,9 +570,7 @@ def makeGraph(self, pipeline: Pipeline, args: SimpleNamespace) -> Optional[Quant
         if args.qgraph:
             # click passes empty tuple as default value for qgraph_node_id
             nodes = args.qgraph_node_id or None
-            qgraph = QuantumGraph.loadUri(
-                args.qgraph, butler.registry.dimensions, nodes=nodes, graphID=args.qgraph_id
-            )
+            qgraph = QuantumGraph.loadUri(args.qgraph, nodes=nodes, graphID=args.qgraph_id)
```

Comment:
Does it hurt to pass the dimension universe through in this call? If not, is ctrl_mpexec supposed to check that the universes are compatible itself?